iati/tests/data/strings.py

```diff
@@ -4,11 +4,12 @@
 
 RAW_HTML = '''
 <p align="justify">
-Lorem ipsum dolor sit amet, accumsan neque lectus ligula mattis netus. A pede posuere. Sem cras torquent, mi lorem augue, risus amet et semper vehicula est. Ipsum vivamus, morbi tincidunt, suspendisse non justo montes, sociis nullam elit metus interdum integer, donec amet nulla consequat praesent. Integer mauris morbi quis sapien, pretium aenean pulvinar diam iaculis eros volutpat, amet turpis non placerat suscipit phasellus lacus, lectus sed sem libero aliquam ligula. Consequat tincidunt vitae proin maecenas habitant lorem, parturient lacinia vehicula a dolor, maecenas lectus interdum arcu lobortis quisque mauris, praesent quis primis, purus aliquam ipsum tristique. Curabitur nec, erat tempora, accumsan et enim consectetuer magnis ligula, justo fermentum lacus imperdiet, eu sed habitasse.
+Lorem ipsum dolor sit amet, accumsan neque lectus ligula mattis netus. A pede posuere. Sem cras torquent, mi lorem augue, risus amet et semper vehicula est. Ipsum vivamus, morbi tincidunt, suspendisse non justo montes, sociis nullam elit metus interdum integer, donec amet nulla consequat praesent. Integer mauris morbi quis sapien, pretium aenean pulvinar diam iaculis eros volutpat, amet turpis non placerat suscipit phasellus lacus, lectus sed sem libero aliquam ligula. Consequat tincidunt vitae proin maecenas habitant lorem, parturient lacinia vehicula a dolor, maecenas lectus interdum arcu lobortis quisque mauris, praesent quis primis, purus aliquam ipsum tristique. Curabitur nec, erat tempora, accumsan et enim consectetuer magnis ligula, justo fermentum lacus imperdiet, eu sed habitasse.'''  # noqa: E501
+RAW_HTML += '''
```

Comment:
Okay, I tried fixing the long string errors a different (but possibly equally unsatisfactory) way!